### ISSUE TITLE
The table on the Mayor main trolly tour day 3 article does not have any scope attributes

### ISSUE THREAD

**Issue by ollie-iterators** [CityOfBoston/Iterators#132]
Screengrab/videos (insert file names here)
2020 08 18, Table has no scope Associations.mp4
[2020 08 18, Table has no scope Associations.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5099036/2020.08.18.Table.has.no.scope.Associations.mp4.zip)

Link
https://d8-stg.boston.gov/calendar/mayor-main-trolley-tour-day-3

Node # (if known)
45361

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.6 > Chrome 84

Trusted Tester Test
14) Tables

Brief Description
The table on the Mayor main trolly tour day 3 article does not have any scope attributes

Steps to reproduce
1)	Go to the Mayor Main trolley tour day 3 article site
2)	00:00 The table on the page "Mayor Main Trolley Tour Day 3" does not have any scope attributes. This is a bug for the Table test on Accessibility.

Desired Result
•	The table on the page has scope attributes

Actual Result
•	The table on the page does not have scope attributes

**Comment by ollie-iterators** [comment]
This also happens to the table on these links:

- https://d8-stg.boston.gov/departments/digital-team/test-page
- https://d8-stg.boston.gov/calendar/mayor-main-trolley-tour-day-1
- https://d8-stg.boston.gov/departments/emergency-medical-services/how-apply-ems-special-event-permit

**Comment by subaha-cob** [comment]
Test on: https://d8-uat.boston.gov/

**Comment by andrebtlima** [comment]
@ollie-iterators is it acceptable to us TH elements with no scope attribute for simple tables that have the headers in the first row or column? Just looking at the WCAG and want to make sure I'm not misinterpreting it.

**Comment by andrebtlima** [comment]
Also @subaha-cob looking on UAT I don't see any additional scope attributes for the tables on the three pages that ollie references in their comment from 11 days ago.

**Comment by david-iterators** [comment]
Hi @andrebtlima and @subaha-cob We have retested this and clarified the tables using a new issue:  https://github.com/CityOfBoston/Iterators/issues/171    Therefore, you can close this current issue and refer to issue #171.  In brief, the tables do now have accessibility information for the cells.  However, the table title (or caption) is not present.  Additional information is added to #171.  Please let us know if you have any questions.  Thanks
